pkgs/by-name/mk/mkclean/package.nix

```diff
@@ -1,45 +1,35 @@
 {
-  dos2unix,
-  fetchurl,
   lib,
   stdenv,
+  fetchurl,
+  cmake,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "mkclean";
-  version = "0.8.10";
-
-  hardeningDisable = [ "format" ];
-  nativeBuildInputs = [ dos2unix ];
+  version = "0.9.0";
 
   src = fetchurl {
-    url = "mirror://sourceforge/matroska/${pname}-${version}.tar.bz2";
-    sha256 = "0zbpi4sm68zb20d53kbss93fv4aafhcmz7dsd0zdf01vj1r3wxwn";
+    url = "mirror://sourceforge/matroska/mkclean-${finalAttrs.version}.tar.bz2";
+    hash = "sha256-L1zcqw4Jtl+f74lJpV7wDuPdcA5LQFDiRdRCNH18w9s=";
   };
 
-  configurePhase = ''
-    dos2unix ./mkclean/configure.compiled
-    ./mkclean/configure.compiled
-  '';
+  nativeBuildInputs = [ cmake ];
 
-  buildPhase = ''
-    make -C mkclean
-  '';
+  hardeningDisable = [ "format" ];
 
-  installPhase = ''
-    mkdir -p $out/{bin,lib}
-    mv release/gcc_linux_*/*.* $out/lib
-    mv release/gcc_linux_*/* $out/bin
+  postInstall = ''
+    install -Dm0755 mkclean/mkclean $out/bin/mkclean
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Command line tool to clean and optimize Matroska (.mkv / .mka / .mks / .mk3d) and WebM (.webm / .weba) files that have already been muxed";
     homepage = "https://www.matroska.org";
-    license = licenses.bsdOriginal;
-    maintainers = with maintainers; [ cawilliamson ];
+    license = lib.licenses.bsdOriginal;
+    maintainers = with lib.maintainers; [ cawilliamson ];
     platforms = [
       "i686-linux"
       "x86_64-linux"
```

Comment:
I don't understand why this was ever set - it builds fine for me on aarch64-linux and the source has notes on building on osx.

Reply:
I agree that a new PR should be open to fix this particular issue.